VideoLocker/src/main/java/org/edx/mobile/view/UserProfileFragment.java

```diff
@@ -249,8 +248,10 @@ private void setAccount(@Nullable final Account account) {
                 } else {
                     viewHolder.bioText.setVisibility(View.VISIBLE);
                     viewHolder.bioText.setText(account.getBio());
+                    bodyBackgroundColor = R.color.white;
```

Comment:
Only on the variable declaration. All it does is warn you if you try to assign something to that variable that isn't a color resource.